pom.xml

```diff
@@ -42,8 +42,8 @@
         <module>model-dropwizard-4</module>
         <module>validation-dropwizard-4</module>
         <module>logging-dropwizard-4</module>
-        <module>queue-java-aws-sdk-v2</module>
         <module>queue-dropwizard-4</module>
+        <module>queue-dropwizard-4-aws-sdk</module>
```

Comment:
Shouldn't we keep `v2` also in the name? "Plain" queue module also has AWS SDK so we might want to distinguish between?